tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_rand.h

```diff
@@ -16,20 +16,24 @@ inline void rand_init(uint seed) {
 }
 
 template <bool APPROXIMATION_MODE>
-inline void rand() {
+inline void rand(float from, float to) {
```

Comment:
nit, randf should sound more correct than rand.

Reply:
I follow torch, c++ API, all of which use rand() to generate random float.